tailwind.config.js

```diff
@@ -158,6 +158,8 @@ module.exports = {
         '-12.5': '-3.125rem',
         '-25': '-6.25rem',
         '-27.5': '-6.875rem',
+        '-29': '-6.563rem',
```

Comment:
Do we need such precision? Can we use `-6.56`?

Reply:
Fixed